sample.env

```diff
@@ -4,7 +4,14 @@ SOLIDITY_VERSION="0.8.13"
 MNEMONIC="deputy annual debris outer baby prefer mammal scene insect obtain parrot length zero detail average nation address depart stem tuition fork rocket topple kidney"
 DEPLOYER=0xff22437ccbedfffafa93a9f1da2e8c19c1711052799acf3b58ae5bebb5c6bd7b
 
-DEPLOYMENT_SALT=0
+MAINNET_DEPLOYMENT_SALT=0
+TESTNET_DEPLOYMENT_SALT=0
+DEVELOP_DEPLOYMENT_SALT=1000
```

Comment:
Oh yes I see that here. I'll set this manually for now and deploy to develop soon